screens/HealthProfessionalScreen.js

```diff
@@ -1,158 +1,145 @@
+        'FieldofPractice':'',
+        'LicenseNumber': '',
+        'RegulatoryBody': ''
         })
-     }*/
-
-
-    // const onHealthPress = async (res) => {
-    //     const data = await firebase.auth().currentUser.uid
-    //     var userDoc = firebase.firestore().collection("users").doc(data).update({
-    //         'FieldofPractice': '',
-    //         'LicenseNumber': '',
-    //         'RegulatoryBody': ''
-    //     })
-
-    //     const obj = {
-
 
-   const onHealthPress = async (res) => {
-   const data = await firebase.auth().currentUser.uid
-        var userDoc = firebase.firestore().collection("users").doc(data).update({
-            'FieldofPractice':'',
-            'LicenseNumber': '',
-            'RegulatoryBody': ''
-        })
         const obj = {
-            FieldofPractice,
-            LicenseNumber,
-            RegulatoryBody,
+        FieldofPractice,
+        LicenseNumber,
+        RegulatoryBody,
         };
-        const usersRef = firebase.firestore().collection('users')
-        usersRef
-            .doc(data)
-            .update(obj)
+        
+        usersRef.doc(data).update(obj)
 
     }
 
+    const HealthProfAlert = () => {
+        alert('Your account will need validation in 2-3 business days');
+        navigation.navigate('HomeScreen')
+      };
+
+
     return (
-        <View style={styles.container}>
+        
+        <KeyboardAvoidingView style={styles.background} behaviour="padding" enabled>
             <Background/>
-            <TouchableOpacity style={styles.menuButton} onPress={() => navigation.openDrawer()}>
-                <MenuIcon />
+            <TouchableOpacity style={styles.menuButton} onPress={()=> navigation.openDrawer()}>
+                <MenuIcon/>
             </TouchableOpacity>
-            <Animatable.View style={styles.drawer} animation="fadeInUpBig"> 
-
-            <View style={{ flexDirection: 'row', alignItems: 'center', marginBottom: screenHeight / 10 }}>
-                <NurseEllieLogo height={75} style={{ flex: 1, marginRight: '5%' }} />
-                <Text style={{ fontFamily: 'roboto-regular', fontSize: 25, }}> {`Account Change:\n Health Professional`}</Text>
-            </View>
-
-                <TextInput style={styles.textInput} placeholder="Field of Practice" autoCapitalize="none"  onChangeText={(text) => setFieldofPractice(text)}
+        <Animatable.View style={styles.drawer} animation="fadeInUpBig">
+        <View style={styles.rowContainer}>
+        <Image style={styles.headerImage} source={require('../assets/android/drawable-mdpi/login-logo.png')} />
+                <Text style={styles.headerFont}> Health Professional</Text>
+                <Text style={styles.headerFont}> Sign Up</Text>
+                <View style={styles.whitePadding}/>
+               
+                <TextInput style={styles.textInput} placeholder="FieldofPractice" autoCapitalize="none"  onChangeText={(text) => setFieldofPractice(text)}
```

Comment:
Can you change the screen header text and image to use SVG and also keep the text and styling of the master's version. 

```suggestion
                    <Animatable.View style={styles.drawer} animation="fadeInUpBig"> 

            <View style={{ flexDirection: 'row', alignItems: 'center', marginBottom: screenHeight / 10 }}>
                <NurseEllieLogo height={75} style={{ flex: 1, marginRight: '5%' }} />
                <Text style={{ fontFamily: 'roboto-regular', fontSize: 25, }}> {`Account Change:\n Health Professional`}</Text>
            </View>

                <TextInput style={styles.textInput} placeholder="Field of Practice" autoCapitalize="none"  onChangeText={(text) => setFieldofPractice(text)}
```